docs/users/subscriptions/index.md

```diff
@@ -12,3 +12,52 @@ links:
+
+### Billing date and subscription renewal
+You can view your account's subscription details, payment history and your next billing date in your account's billing settings at {% mentionPost "/docs/users/subscriptions/upgrade-subscription" %}
+
+## Billing settings
+You can manage your billing settings, such as subscription plans and payment methods in your account's billing settings at {% mentionPost "/docs/users/subscriptions/upgrade-subscription" %}
+
+## Overages
+At the moment Devopness does not allow usage-based billing, so user projects cannot consume more credits than the amount included in the user’s active subscription plan.
+
+There will be no surprises in your monthly or annual billing, as Devopness will charge you upfront for the included amounts of credits in your subscription plan.
+
+You can see the amount of credits included in your account's billing settings at Billing & Plans 
+
+## Usage limits
+There are some limits when using Devopness actions, depending on the subscription plan.
+These limits are subject to change every time our plans are updated. For details, see our pricing page https://www.devopness.com/pricing/
+
+Below you have a table with the current applied limits, per plan, followed by an explanation on the impact of each limit:
+|                                                             | Free       | Standard             |
+|:-------------------------------------------------------------|:-----------|:---------------------|
+| **Max parallel actions, per server**                         | 1          | 10                   |
+| **Max parallel actions, per pipeline**                       | 1          | 10                   |
+| **Max amount of time an action step log remains accessible** | 24 hours   | 90 days              |
+| **Max execution time, per action step**                      | 10 minutes | 15 minutes           |
+| **Max API requests, per minute**                             | 30         | 120                  |
+
+- **Max parallel actions, per server**: Users can control how many actions can run simultaneously on each server, so the server resources are used the way that is most suitable for each user environment
+
+- **Max parallel actions, per pipeline**: User pipelines may run custom action steps that could lead to race conditions if executed concurrently. Devopness allows user to define which pipelines can have multiple actions executing it at the same time and how many parallel actions are allowed to be started running that pipeline.
+
+- **Max execution time, per action step**: Each pipeline step executed by an action can run for up to 15 minutes of execution time. If an action step reaches this limit, the step is moved to the “failed” status and the action fails to complete.
+
+## Usage policy
+In addition to the usage limits, you must ensure that you use Devopness within our {% mentionPost "/docs/legal/terms-of-service/" %}
```

Comment:
The current implementation of `mentionPost` does not handle paths terminated with '/', causing a build error

```suggestion
In addition to the usage limits, you must ensure that you use Devopness within our {% mentionPost "/docs/legal/terms-of-service" %}
```